src/redux/conference-schedule.js

```diff
@@ -0,0 +1,136 @@
+var keyMirror = require('keymirror');
+var api = require('../mixins/api.jsx').api;
+
+var Types = keyMirror({
+    SET_SCHEDULE: null,
+    SET_SCHEDULE_FETCHING: null,
+    SET_SCHEDULE_ERROR: null
+});
+
+module.exports.scheduleReducer = function (state, action) {
+    if (typeof state === 'undefined') {
+        state = {
+            chunks: [],
+            day: ''
+        };
+    }
+    switch (action.type) {
+    case Types.SET_SCHEDULE:
+        return action.schedule;
+    case Types.SET_SCHEDULE_FETCHING:
+        return state;
+    case Types.SET_SCHEDULE_ERROR:
+        return state;
+    default:
+        return state;
+    }
+};
+
+module.exports.setSchedule = function (schedule) {
+    return {
+        type: Types.SET_SCHEDULE,
+        schedule: schedule
+    };
+};
+
+module.exports.setScheduleFetching = function () {
+    return {
+        type: Types.SET_SCHEDULE_FETCHING,
+        fetching: true
+    };
+};
+
+module.exports.setScheduleError = function (error) {
+    return {
+        type: Types.SET_SCHEDULE_ERROR,
+        error: error
+    };
+};
+
+module.exports.startGetSchedule = function (day) {
+    return function (dispatch) {
+        dispatch(module.exports.setScheduleFetching());
+        dispatch(module.exports.getDaySchedule(day));
+    };
+};
+
+/**
+ * Gets the schedule for the given day from the api
+ * @param  {String} day  Day of the conference (Thursday, Friday or Satrurday)
+ *
+ *  @return {Object}     Schedule for the day, broken into chunks
```

Comment:
Thanks for updating it, that's clearer. Don't forget to update this docstring too.